host_cleanup.sh

```diff
@@ -42,10 +42,9 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
     -i vm-setup/inventory.ini \
     -b -v vm-setup/teardown-playbook.yml
 
-if [ "$USE_FIREWALLD" == "False" ]; then
+if [ "$USE_FIREWALLD" = "false" ]; then
  ANSIBLE_FORCE_COLOR=true ansible-playbook \
-    -e "{use_firewalld: $USE_FIREWALLD}" \
-    -e "external_subnet_v4: ${EXTERNAL_SUBNET_V4}" \
+    -e "use_firewalld=${USE_FIREWALLD}" \
```

Comment:
this condition does not make sense for me if we are checking in ansible playboot use firewalld why we need to check here ?

Reply:
I parameterized cleaning firewalld as well https://github.com/metal3-io/metal3-dev-env/pull/1179

Reply:
Okay, your PR does other stuff too, so let's remove this firewall condition here and then you don't need to deal with @tuminoid 's comments related to the argument formatting.